opennms-doc/guide-admin/src/asciidoc/text/provisioning/examples.adoc

```diff
@@ -291,6 +291,35 @@ Add a policy parameter and choose the _sysObjectId_ key with a value `~^\.1\.3\.
 .Example: Node Category setting policy
 image:../images/provisioning/00020.jpeg[]
 
+.Script Policy
+
+This policy allows to use _Groovy_ scripts to modify provisioned node data. These scripts have to placed in the
```

Comment:
Please follow the one sentence per line guidelines for documentation.

Comment on the last sentence:
> These scripts have to *be* placed in the `${OPENNMS_HOME}/etc/script-policies` directory.